src/niraj/foo.ts

```diff
@@ -0,0 +1,2 @@
+export const sum = (...a: number[]): number =>
```

Comment:
Can we provide more meaningful filename instead of foo.ts? ex: sum.ts etc
Same for variable names as well